### ISSUE TITLE
Blaze campaign notification says the ad cost $0. It didn't

### ISSUE THREAD

**Issue by dzver** [Automattic/wp-calypso#93942]
### Quick summary

I got a notification a few days after the campaign was over. The notification says the campaign costs $0 but when I click it, it says it costs $15

The notification shows $0 on both Desktop and Mobile

Also, the date is not human readable.

### Steps to reproduce

1. Buy a campaign
2. Wait a week for the campaign to be over
3. Observe this notification

### What you expected to happen

The $0 to be the actual amount of the campaign ($15)
The date should be human readable

### What actually happened

Saw $0

<img width="800" alt="Screenshot 2024-08-27 at 12 06 45" src="https://github.com/user-attachments/assets/e1c3e37a-46ef-4957-a635-05bd63de0605">

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by jeherve** [comment]
That date display is pretty ugly too, it may be worth looking into as well:

<img width="329" alt="image" src="https://github.com/user-attachments/assets/41941f8a-177e-4f34-abf4-f0c31f1faae2">

**Comment by jeherve** [comment]
cc @sbarbosa Is that something y'all could look into?

**Comment by sbarbosa** [comment]
Yes, that notification originates in our DSP server. I will create a ticket in our repo to track this problem. 
Thanks for the ping.

**Comment by sbarbosa** [comment]
Let me tag @enejb  here. I locate where we build the notification text (in WPCOM), but the other screen where we show the spending, I think is inside the apps.

**Comment by sbarbosa** [comment]
The notification should be fixed now. This issue can be re-tested and closed.

**Comment by dzver** [comment]
Thanks!

Do you have any screenshots of the fixed email and notification? Or a link?

**Comment by sbarbosa** [comment]
Sure, here are the screenshots:
![Screenshot 2024-09-19 at 10 01 45 AM](https://github.com/user-attachments/assets/8c7005a6-0bef-4c47-9b9f-7b757e4b4657)
![Screenshot 2024-09-19 at 10 01 39 AM](https://github.com/user-attachments/assets/7c965b71-9be9-44f1-a468-2bf223e0f539)
![Screenshot 2024-09-19 at 10 01 30 AM](https://github.com/user-attachments/assets/07c15590-2869-4b50-b125-18136aee5513)

**Comment by dzver** [comment]
It can benefit from some formatting. 

September 14, 2024, 13:22 UTC - I wonder how many users would know what UTC is? Can we display their date?

2024-09-14 13:22:15 - that's clearly UTC as well. Also, oddly specific. The format is great for engineers, one of the most readable date formats. However, non-engineers might be surprised to see it.

USD 2.14 is not how we normally display prices. We usually display them with the currency symbol and internationalized, so it would be $2.14 in your case.

**Comment by sbarbosa** [comment]
Thanks! I will update an reopen our ticket with this feedback.

**Comment by dzver** [comment]
Thanks!

**Comment by dzver** [comment]
Got it. You can ask the Payments Shilling team about that, I'm only pointing out the discrepancy.

**Comment by dzver** [comment]
<img width="809" alt="Image" src="https://github.com/user-attachments/assets/da9f2595-0897-4be4-968e-c4e5069c188a" />

The issue is still reproducible on production.

@javiolmo89 @sbarbosa

**Comment by sbarbosa** [comment]
Thanks for the ping. I will reopen the ticket on our side. 

This happens because the spend field is calculated using the campaign orders. And for this specific case, the mail was sent 1 day before the order came to our system.

**Comment by dzver** [comment]
<img src="https://github.com/user-attachments/assets/ddc98f1c-a12f-41d0-9c2f-3ebe1abbdfbe" width="40%" />

**Comment by dzver** [comment]
<img width="411" alt="Image" src="https://github.com/user-attachments/assets/7e3078cc-c6c5-4feb-b05e-245e63964896" />

**Comment by timur987** [comment]
It seems I can't continue working on the issue (at least for now).
Duplicated the issue in our core repository for visibility: `3448-ghe-Tumblr/a8c-dsp`
